src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -80,6 +81,21 @@ public static Address parseAddress(String address) throws ParseException {
         return new Address(trimmedAddress);
     }
 
+    /**
+     * Parses a {@code String lastSeen} into an {@code LastSeen}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code lastSeen} is invalid.
+     */
+    public static LastSeen parseLastSeen(String lastSeen) throws ParseException {
+        requireNonNull(lastSeen);
+        String trimmedLastSeen = lastSeen.trim();
+        if (!LastSeen.isValidDate(trimmedLastSeen)) {
+            throw new ParseException(LastSeen.MESSAGE_CONSTRAINTS);
+        }
+        return new LastSeen(lastSeen);
```

Comment:
Hi @O-Darren,  just to double confirm if you intended to `return new LastSeen(lastSeen)` or was it supposed to be `return new LastSeen(trimmedLastSeen)`

Reply:
Intended to return trimmedLastSeen, thanks for the catch